### PR TITLE
Cleanup for GVL

### PR DESCRIPTION
**Pull request by mike-plummer**
Adding Formik example slides
Removing Redux API examples, expanding react-redux content instead

### CODE REVIEW

**Comment by DSchau** [comment]
Deploy preview for *react-redux-workshop* ready!

Built with commit 678e5763eb741ccba2a504b627574d37da0cfa64

https://deploy-preview-78--react-redux-workshop.netlify.com

File: slidedeck/src/slides/09-special-topics/05-async-action-libraries.pug [hunk 1]
```diff
@@ -18,6 +18,7 @@ section
     li Used to manage application side effects (asynchronous server calls, accessing local storage, etc.)
     li Implemented as Redux middleware
     li Uses ES6 Generators to make async code easier to read, manage, and test
+  a(href="https://mike-plummer.github.io/redux-sagas-presentation/") Check this out for more info
```
Comment (JakePartusch) [inline slidedeck/src/slides/09-special-topics/05-async-action-libraries.pug]: Shameless plug ;)

File: slidedeck/src/slides/09-special-topics/10-testing.pug [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <MemoryRouter>
+          <MyAwesomeComponent />
+        </MemoryRouter>
+      </Provider>
+    );
+
+section
+  h3 Peeling those layers back can make testing a lot simpler
+  script(type="text/plain", class="language-jsx").
+    export const _MyAwesomeComponent = (props) => ...;
```
Comment (JakePartusch) [inline slidedeck/src/slides/09-special-topics/10-testing.pug]: Why the underscore here?

Comment (mike-plummer) [reply slidedeck/src/slides/09-special-topics/10-testing.pug]: Was trying to figure out a way to more effectively manage the "named vs default" export issue when testing wrapped/unwrapped versions of components

File: slidedeck/src/slides/09-special-topics/05-async-action-libraries.pug (hunk 1 above)
Comment (mike-plummer) [reply slidedeck/src/slides/09-special-topics/05-async-action-libraries.pug]: Hey, gotta pump up my Github star count somehow...

**Pull request merged**